assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -4269,46 +4269,37 @@ tests:
           model
             schema 1.1
           type user
-          type folder
+          type folder1
+          type folder2
             relations
-              define owner: [user]
-              define writer: [user,user:*] or owner
+              define viewer: [user]
           type document
             relations
-              define can_read: writer from parent
-              define parent: [document,folder]
-              define viewer: [user,user:*]
+              define viewer: viewer from parent
+              define parent: [folder1,folder2]
         tuples:
-          - user: user:anne
-            relation: owner
-            object: folder:a
-          - user: folder:a
-            relation: parent
-            object: document:c
-          - user: document:c
+          - user: folder1:x
             relation: parent
-            object: document:d
+            object: document:y
         checkAssertions:
           - tuple:
               user: user:anne
-              relation: can_read
-              object: document:c
-            expectation: true
+              relation: viewer
+              object: document:y
+            expectation: false
```

Comment:
I modified this test instead of creating a new one because: the old test provides little value. If there is a truthy path available, we may not exercise the buggy path under test.

The desired behavior is _even if there are no truthy paths available_ we should _not_ error.

Reply:
Should we add some assertions to verify the truth case (i.e., folder2 path?)